compiler/rustc_span/src/symbol.rs

```diff
@@ -1901,6 +1901,13 @@ impl fmt::Display for Symbol {
     }
 }
 
+// takes advantage of `str::to_string` specialization
+impl ToString for Symbol {
+    fn to_string(&self) -> String {
+        return self.as_str().to_string();
```

Comment:
```suggestion
        self.as_str().to_string()
```

Reply:
I need to take a mental break after switching from Python.